common4j/src/main/com/microsoft/identity/common/java/controllers/CommandDispatcher.java

```diff
@@ -550,6 +550,8 @@ private static CommandResult executeCommand(@SuppressWarnings(WarningType.rawtyp
             if (baseException instanceof UserCancelException) {
                 commandResult = CommandResult.ofNull(CommandResult.ResultStatus.CANCEL,
                         correlationId);
+            } else if (baseException.getErrorCode().equals(ClientException.OUT_OF_MEMORY)) { // If we receive an out of memory error
+                command.getParameters().getPlatformComponents().getPlatformUtil().handleShutdownForOutOfMemoryError(baseException, TAG);
```

Comment:
>command.getParameters().getPlatformComponents().getPlatformUtil().handleShutdownForOutOfMemoryError(baseException, TAG); [](http://example.com/codeflow?start=16&length=120)

I think there could be OOM errors even before command dispatcher.

Reply:
What is the earliest common point to do this? I'm not sure if we can do msalbroker result adapter, IDK if that code is ran as part of broker process or msal process.

Reply:
Resolving as i'm no longer crashing broker